### PR TITLE
Dynasties + Socotra

### PR DESCRIPTION
**Pull request by IhateTrains**
* made-up c_socotra history up to 600.1.1
* added missing dynasties + minor character fixes
* code cleaning

### CODE REVIEW

File: WTWSMS/common/dynasties/WtWSMS_vanilla_dynasties.txt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	culture="panjabi"
+	religion="hindu"
+	coat_of_arms=
+	{
+		data=
+		{
+0 2 0 0 0 0 0 			}
+		religion="hindu"
+	}
+}
```
Comment (loup99) [inline WTWSMS/common/dynasties/WtWSMS_vanilla_dynasties.txt]: I would rather add them to the vanilla file.

File: WTWSMS/common/dynasties/erik_dynasties.txt [hunk 2]
```diff
@@ -77,4 +77,14 @@
...
+}
+1100004 = { # for made-up Gallus, governor of Thrace
+	name="Gallus"
+	culture = thraco_roman
+	religion = latin_solar_pagan_reformed
+}
+11000111 = { # for made-up Eucleidas, governor of Thrace
+	name="Eucleidas"
+	culture = tsakonian
+	religion = mithraic_pagan
```
Comment (loup99) [inline WTWSMS/common/dynasties/erik_dynasties.txt]: Why did you add these here?

File: WTWSMS/common/dynasties/WtWSMS_vanilla_dynasties.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/dynasties/WtWSMS_vanilla_dynasties.txt]: Moved to 00_dynasties.txt

File: WTWSMS/common/dynasties/erik_dynasties.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Eucleidas is in zusk characters, so I thought it's a fitting place

Comment (loup99) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Yes, I understand. But isn't the intent to have random dynasties?

Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Euclidas has a defined dynasty ID, but that dynasty was missing, so I added it

Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: This is Eucleidas:
11000111 = {
	name="Eucleidas"
	# Make-up Governor of Thrace
	religion=mithraic_pagan
	culture=tsakonian
	dynasty=11000111
	414.1.1={
		birth = yes
	}
	464.1.1={
		death = yes
	}
}

Comment (loup99) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Yes, but isn't the effect to get a random one?

**Comment by IhateTrains** [comment]
I have resolved the conflicts.

File: WTWSMS/common/dynasties/erik_dynasties.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Will check in game.

Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: An Eucleidasid dynasty is generated properly, but is the "dynasty=" line required to generate?

Comment (loup99) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: I don't know, but you can test.

Comment (IhateTrains) [reply WTWSMS/common/dynasties/erik_dynasties.txt]: Tested, it's not needed to generate a dynasty. But given that this character is in 395-476 period and he may get a family in the future, I still think defining a dynasty is a better solution.
